styles/status.cafe/catppuccin.user.css

```diff
@@ -0,0 +1,194 @@
+        background-color: @base;
+
+        body {
+            background-color: @base;
+            color: @text;
+        }
+        
+        textarea {
+            background-color: @mantle;
+            border: 1px solid @overlay2;
+            border-radius: 2px;
+            padding: 3px;
+        }
+        
+        input {
+            background-color: @crust;
+            border: 1px solid @overlay2;
+            border-radius: 4px;
+            padding: 2px 5px 2px 5px;
+        }
+        
+        input#homepage, input#email, input#picture {
+                background-color: @mantle !important;
+            }
+
+        .flash {
+            background-color: @green;
+        }
+
+        /*accent color links*/
+        & when (@colors = 1) {
+            a:not(.flash a) {
+                color: @accent-color;
+            }
+
+            a:visited:not(.flash a) {
+                color: @overlay0;
+            }
+
+            /* visited colors*/
+            & when (@accent = rosewater), (@accent = flamingo), (@accent = lavender) {
+                a:visited:not(.flash a) {
+                    color: @pink;
+                }
+            }   
+            & when (@accent = pink) {
+                a:visited:not(.flash a) {
+                    color: @mauve;
+                }
+            }
+            & when (@accent = mauve), (@accent = teal),(@accent = sky) {
+                a:visited:not(.flash a) {
+                    color: @blue;
+                }
+            }
+            & when (@accent = red), (@accent = maroon) {
+                a:visited:not(.flash a) {
+                    color: @peach;
+                }
+            }
+            & when (@accent = peach) {
+                a:visited:not(.flash a) {
+                    color: @red;
+                }
+            }
+            & when (@accent = yellow) {
+                a:visited:not(.flash a) {
+                    color: @maroon;
+                }
+            }
+            & when (@accent = green), (@accent = sapphire) {
+                a:visited:not(.flash a) {
+                    color: @teal;
+                }
+            }
+            & when (@accent = blue) {
+                a:visited:not(.flash a) {
+                    color: @sky;
+                }
+            }
+            & when (@accent = teal) {
+                a:visited:not(.flash a) {
+                    color: @blue;
+                }
+            }
+        }
+
+        & when (@lookup = latte){
+            & when (@accent = sapphire) {
+                a:visited:not(.flash a) {
+                    color: @lavender;
+                }
+            }
+            .flash {
+                color: #c9edd1;
+                a {
+                    color: @blue;
+                }
+            }
+        } 
+        
+        /*never breaking columns*/
+        /*creds to scarecat.neocities.org*/
+        & when (@break = 1) {
```

Comment:
Since this is an opinionated and non-color change, we can't include this in the user style. Users can install it through another user style instead.